.github/DISCUSSION_TEMPLATE/sandbox.yml

```
@@ -0,0 +1,52 @@
+name: "Sandbox"
+description: "Use this form to raise a query, request, or report a bug in the sandbox environment."
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Use this form to raise a query, request, or report a bug. 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)